module_utils/helpers.py

```diff
@@ -7,7 +7,7 @@
 
 
 class ModuleRes:
-    def __init__(self, success, msg=None, changed=False, meta=None, warnings=[]):
+    def __init__(self, success, msg=None, changed=False, meta=dict(), warnings=[]):
```

Comment:
It's a bad practice to use mutable values as defaults. Use `None` value instead.
https://docs.python-guide.org/writing/gotchas/

Reply:
Please, see my fix.